torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -545,6 +545,35 @@ def ReplaceMoreTtManually(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
                             args[0],
                             delete_user_cb=lambda node: node != args[0],
                         )
+            if node.target == torch.ops.aten.slice.Tensor:
+                # Returns the new node or `None` if unsupported
+                def callback(tensor, dim, start, end, step=1):
+                    input_size = tensor.meta["val"].size()
+
+                    if step != 1 or dim >= len(input_size):
+                        return None
+
+                    # NOTE(jdh8): is there a way to skip a no-op?
```

Comment:
@kevinwuTT is there a way to skip an op if its a no-op?

Reply:
Returning `tensor` seems fine here. The following code will automatically replace the uses of the `aten.split` op to point to the `arg[0]` tensor, which effectively removes this op.

https://github.com/tenstorrent/pytorch2.0_ttnn/blob/a5ce2a677f6ea9c6ba704c21db1c771bff03ef8a/torch_ttnn/passes/lowering/to_tt_pass.py#L592-L596